addons/l10n_ec/data/account_chart_template_data.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data noupdate="1">
-        <function model="account.chart.template" name="try_loading">
-            <value eval="[ref('l10n_ec.ec_chart_template')]"/>
-        </function>
-    </data>
+	<data noupdate="1">
+		<record id="l10n_ec_ifrs" model="account.chart.template">
+			<field name="name">Ecuador NIIFs por Trescloud (Conforme Superintendencia de Compañías)</field>
+			<!-- <field name="transfer_account_id" ref="l10n_ec_ifrs_liquidity_transfer"/>  -->
+			<field name="currency_id" ref="base.USD"/>
+			<field name="code_digits">1</field>
```

Comment:
I'll need to check.

Reply:
It might be a good idea to change to 4, a typicial chart of accounts in Ecuador might have account like:
1 first level, handled in account.group
11 second level, handled in account.group
1101 third level, handled in account.account

But given that now we have the account.groups it might be changed from 1 to "4" (the third level)